README.md

```diff
@@ -17,4 +17,85 @@ Sample output:
     "percentChange":-90.54688681639148
   }
 }
-```
\ No newline at end of file
+```
+
+### `/public/trades`
+Historical trades
+Return historical trades executed. Trades returned in ascending order. Option available to query historical trades based on tradeID, default is
+to return latest 100 trades
+
+#### Params
+
+|Name        | Type                 | Example                                             | Description
+|------------|----------------------|-----------------------------------------------------|----------------------------
+|pair        | String               | ETH_EOS                                             | 
```

Comment:
Can you add both here in description and above that direction doesn't matter (and make sure it does not).

Reply:
Call result: throw error on EOS_ETH